src/scenes/OrgRouter/NavBar.tsx

```diff
@@ -256,9 +291,10 @@ export default function NavBar({
           )}
         </SubNavbar>
       </NavItem>
-      {!hasDeliverables && reportsMenu && (
+      {!hasDeliverables && ((isEnabled('Participant Experience') && modulesMenu) || reportsMenu) && (
         <>
-          <NavSection />
+          <NavSection title={strings.ACCELERATOR.toUpperCase()} />
+          {isEnabled('Participant Experience') ? modulesMenu : null}
```

Comment:
Can you set the flag check to a variable? `{featureFlagParticipantExperience && modulesMenu}`